go-controller/pkg/config/config.go

```diff
@@ -163,6 +163,7 @@ type KubernetesConfig struct {
 	MetricsEnablePprof bool   `gcfg:"metrics-enable-pprof"`
 	OVNEmptyLbEvents   bool   `gcfg:"ovn-empty-lb-events"`
 	PodIP              string `gcfg:"pod-ip"`
+	NoHostSubnetNodes  string `gcfg:"no-hostsubnet-nodes"`
```

Comment:
@JacobTanenbaum add a:

```
	NoHostSubnetNodes  *metav1.LabelSelector
	RawNoHostSubnetNodes  string `gcfg:"no-hostsubnet-nodes"`
```

and then run the ParseToLabelSelector() bit (into the final NoHostSubnetNodes eg not-raw one) from buildKubernetesConfig() so we can hard-error on startup if hte label selector is malformed.

Reply:
fixed